gemfiles/faraday-0.16/Gemfile

```
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+
+# Specify your gem's dependencies in sbpayment.gemspec
+gemspec path: File.expand_path('../../..', __FILE__)
+
+gem 'faraday', '~> 0.16.0'
```

Should we use fixed versioning?

```suggestion
gem 'faraday', '0.16.0'
```

---

I didn't care about the patch version, but it sounds good.